populator/nosql/couchbase-ftest/src/test/java/org/jboss/arquillian/populator/nosql/couchbase/CouchbaseTest.java

```diff
@@ -0,0 +1,41 @@
+package org.jboss.arquillian.populator.nosql.couchbase;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.document.JsonDocument;
+import com.couchbase.client.java.document.json.JsonObject;
+import org.arquillian.cube.HostIp;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.populator.nosql.api.NoSqlPopulator;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Arquillian.class)
+public class CouchbaseTest {
+
+    @HostIp
+    private String hostIp;
+
+    @ArquillianResource
+    @Couchbase
+    NoSqlPopulator populator;
+
+    @Test
+    public void should_find_books() {
+        populator.forServer(hostIp, 0)
+                .withDatabase("travel-sample")
+                .usingDataSet("/airlines.json")
```

Comment:
Well actually this is resolved from classpath relative to `CouchbasePopulatorService` https://github.com/arquillian/arquillian-extension-persistence/pull/61/files#diff-9862b7b39e4c1669a9a8490a16921999R97 so the slash is necessary. Maybe we could do something like if not starting with `/` then we append automatically.